server/src/test/java/com/tencent/rss/server/MockChecker1.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.server;
+
+class MockChecker1 extends Checker {
```

Comment:
Why do we need a separate MockChecker classes named MockChecker1 and MockChcker2 ? And this should be change to more a meaningful name.

Reply:
This is test.

Reply:
The coding style should be standardized in test code too, especial the class and classname.

Reply:
You could add a class named MockChecker and create different instance with different behaviour by constructor conf or just two class named MockCheckerHealthy and MockCheckerUnhealthy, don't use XXX1, XXX2 as a class name.